foxglove/cmd/utils.go

```diff
@@ -228,3 +230,12 @@ func maybeConvertToRFC3339(timestamp string) (string, error) {
 	}
 	return parsed.Format(time.RFC3339), nil
 }
+
+func AuthIsApiKey() (bool, error) {
+	token := viper.Get("bearer_token")
+	if token == nil {
+		dief("Auth is not setup. Run `foxglove auth login` or `foxglove auth configure-api-key` to continue.")
+	}
+	stringifiedToken := fmt.Sprintf("%v", token)
+	return strings.HasPrefix(stringifiedToken, "fox_sk_"), nil
```

Comment:
I think we should treat these as opaque IDs. Maybe we start writing an `auth_type` key to the config in these cases, and we can read that value instead of the token itself.